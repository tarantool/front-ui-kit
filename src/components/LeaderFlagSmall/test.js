import React from 'react';
import renderer from 'react-test-renderer';

import { LeaderFlagSmall } from './index';

it('LeaderFlagSmall renders correctly', () => {
  const tree = renderer.create(<LeaderFlagSmall />).toJSON();

  expect(tree).toMatchSnapshot();
});
