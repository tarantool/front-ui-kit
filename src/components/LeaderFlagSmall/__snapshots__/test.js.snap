// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LeaderFlagSmall renders correctly 1`] = `
.emotion-0 {
  width: 42px;
  height: 8px;
}

<svg
  className="emotion-0"
  viewBox="0 0 42 8"
>
  <use
    xlinkHref="#flagSmall"
  />
</svg>
`;
